app/bundles/UserBundle/Event/AuthenticationEvent.php

```diff
@@ -116,7 +116,7 @@ public function setToken($service, TokenInterface $token): void
     {
         $this->token                 = $token;
         $this->authenticatingService = $service;
-        $this->isAuthenticated       = $token->isAuthenticated();
+        $this->isAuthenticated       = null === $token->getUser();
```

Comment:
Shouldn't this condition not equal NULL?

Reply:
@kuzmany Good find! Fixed.